app.json

```diff
@@ -13,6 +13,7 @@
 		"test": {
 			"addons": ["heroku-postgresql:in-dyno"],
 			"buildpacks": [
+				{ "url": "https://github.com/heroku/heroku-buildpack-apt.git" },
```

Comment:
We needed to explicitly add the apt buildpack to both review and test apps to get the build working since `validate-with-xmllint` (a dependency of `@pubpub/deposit-utils`) checks for `xmllint` on PATH on postinstall.